### PR TITLE
Wordnote Frame with WordCards Count

### PR DESCRIPTION
**Pull request by akanex115**
# Background
(https://github.com/ajktown/wordnote/issues/134)

## TODOs
- [x] Make sure how many word cards are available in the view

## Checklist (Developers)
- [x] Make this PR as a draft first
- [x] Title is checked
- [x] Background & TODOs are filled
- [x] Assignee is set
- [x] Labels are set
- [x] Project is created & linked, if multiple PRs are associated to this PR
- [x] Appropriate issue(s) is(are) linked to this PR under `development`
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

**Review by mlajkim — COMMENTED** [review]
Nice.
Now where do you want to see the number?

**Comment by akanex115** [comment]
Change the position of numbers and add units.
Handled in [b50f8ab](https://github.com/ajktown/wordnote/pull/157/commits/b50f8abacee465157a525bae799ad085d8ab900b)

File: src/components/organism_word_card_chunk/index.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const WordCardsChunk: FC = () => {
   return (
     <StyledSuspense>
       <WordCardChunkSearchFound />
+      {wordIds.length} word cards
```
Comment (mlajkim) [inline src/components/organism_word_card_chunk/index.tsx:25]:
```suggestion
      <Typography variant="caption">
        {wordIds.length} word cards
      </Typography>
```

Use typography to make the font style consistent.
https://mui.com/material-ui/react-typography/#usage

Typography supports many font variants
@akanex115  TODOs
- [ ] Test different variants in your own machine
image

**Review by mlajkim — COMMENTED** [review]
first review

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**